common/djangoapps/student/migrations/0047_add_has_profile_image_boolean.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
```

Comment:
We will need to figure how to handle the fact that we have "multiple 47" migration files.